packages/react/src/hooks/actions/shared/types.ts

```diff
@@ -39,9 +39,9 @@ export interface UseDataStoreActionOptions<Model extends PersistentModel> {
   schema?: Schema;
 }
 
-export function isAlreadyTyped<Model>(
+export function isAlreadyTyped<Model extends PersistentModel>(
   fields: DataStoreActionFields<Model>,
-  schema: Schema
+  schema?: Schema
```

Comment:
Feel like this is a silly question, but this change now allows `isAlreadyTyped` (side note: maybe we can rename) to be called without the `schema` param, which will always return `true` correct? 

Just want to make sure that this is intentional, the change has no impact on current usage (in particular since this ultimately gets used by Studio) and is a change we definitely want. 

Last thing - know you did not write the code initially, but it might be nice to leave some comments on how this should be used as well since this function is surprisingly opaque between its name and logic